dictionaries/software-terms/src/software-terms.txt

```diff
@@ -1704,3 +1704,4 @@ zip
 zlib
 ZPool
 ZPools
+Horovod
```

Comment:
This is a software tool, not a term.

Please add it to `dictionaries/software-terms/src/software-tools.txt` instead of `software-terms`.